site/gatsby-site/src/components/entities/EntitiesTable.js

```diff
@@ -276,3 +279,6 @@ export default function EntitiesTable({ data, className = '', ...props }) {
 
   return <Table table={table} className={className} {...props} />;
 }
+
+var getResponseCount = (responses) =>
```

Comment:
Nit: can you use `const` instead? It is probably the only function in the entire app using `var` :D

Reply:
Done. Mostly for my own future reference, here's what I was thinking:

I like hoisting for functions, since it can allow you to put the most important information at the top. E.g.

```javascript
const importantThing = boringEscapeLogic(importantLogic(parameter));

function importantLogic(e) { ... }

function boringEscapeLogic(e) { ... }
```

I was using `var` because I thought it allowed me to get the same hoisting behavior as `function` but with arrow syntax, but I read more about it and realize that's not true. `var` allows use before declaration without a syntax error, but the value is undefined.

```javascript
// This does not work
const importantThing = boringEscapeLogic(importantLogic(parameter));

var importantLogic = (e) => { }

var boringEscapeLogic = (e) => { }
```

It worked because I didn't actually need hoisting here since the call is contained in another function.

```javascript
// This does work
const getImportantThing = () => boringEscapeLogic(importantLogic(parameter));

var importantLogic = (e) => { }

var boringEscapeLogic = (e) => { }

getImportantThing(parameter);
```

Reply:
Interesting! And that's why I don't like it, it leads to essays about JS quirks 😆 

And I think that's what ECMA tried to get away from when they introduced `let` and `const`, but that's for another essay too 🤣